ui/src/components/PinPreview.tsx

```diff
@@ -72,7 +72,7 @@ export function PinPreview({
           Read Full Story
         </StyledButton>
         <CardDetailText>
-          {author} • {date}
+          {author} • {date.substring(0,4)}
```

Comment:
So that only shows the year - but I think we're changing this in the DB anyways?

Reply:
yeah we will just have a year as a field in database